ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -77,25 +77,20 @@ export default function OnboardingMetametrics() {
       className="onboarding-metametrics"
       data-testid="onboarding-metametrics"
     >
-      <Typography
-        variant={TypographyVariant.H2}
-        align={TEXT_ALIGN.CENTER}
-        fontWeight={FONT_WEIGHT.BOLD}
+      <Text
+        variant={TextVariant.headingLg}
+        as="h2"
+        align={TextAlign.Center}
```

Comment:
I believe you want `textAlign` here this is what is being passed down to the html element. docs: https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-text--text-align-story#text-align
```suggestion
        textAlign={TextAlign.Center}
```